### PR TITLE
Core/Spells: Update DR's for WoD

### PR DESCRIPTION
**Pull request by Zedron**
thx @fatalaim for doing some of the spells

### CODE REVIEW

**Comment by Goatform** [comment]
Try to fi that too, I thinks that it should be fixed in SpellMgr.cpp:
Changelog 5.1.0: The durations of all disarm effects have been reduced to 8 seconds.

**Comment by Aaron126** [comment]
Disarms were removed in wod

**Comment by Goatform** [comment]
Ohh my bad, I didn't found that on changelog list o.O

**Comment by Aaron126** [comment]
This can be merged whenever, everything is working as intended as far as i can tell. commits have been squashed. if anyone has any changes they think should be made I'm open to suggestions

**Comment by Shauren** [comment]
Core/Players: ...[bla]...
we kind of have a standard for writing commit messages (you can see it by just opening commit listing https://github.com/TrinityCore/TrinityCore/commits/6.x)

File: src/server/game/Miscellaneous/SharedDefines.h [hunk 1]
```diff
@@ -4313,7 +4313,7 @@ enum AiReaction
 // Diminishing Returns Types
 enum DiminishingReturnsType
 {
-    DRTYPE_NONE         = 0,                                // this spell is not diminished, but may have limited it's duration to 10s
+    DRTYPE_NONE         = 0,                                // this spell is not diminished, but may have it's duration limited
```
Comment (Kinzcool) [inline src/server/game/Miscellaneous/SharedDefines.h]: Actually, still wrong: it's -> its

Comment (Aaron126) [inline src/server/game/Miscellaneous/SharedDefines.h]: LOL such a gay fish

**Comment by Aaron126** [comment]
aha, thx @Shauren. fixed. I missed that when i squashed the commits. and.. thx @Kinzcool i guess -_-

**Pull request merged**